### PR TITLE
Expose an endpoint for LoadTesting

### PR DESCRIPTION
**Pull request by kajan-bolt**
# Description
This PR allows devs to easily setup a test store and expose an endpoint that triggers the part of the plugin that is responsible for turning a cart on M1 into a bolt order. This allows us to load test a part of the plugin that is not directly exposed by an endpoint. 

Note: The actual load tests will be included in a separate PR (https://github.com/BoltApp/bolt-magento1/pull/545)

Fixes: https://app.asana.com/0/941895179897714/1149771740841853/f

(Optional: Add a changelog by writing a comment after "#changelog" on the next line)

\#changelog

# Type of change

- [ ] Bug fix (change which fixes an issue)
- [ x ] New feature (change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?
Please validate that you have tested your change in at least one of the following areas:

- [ ] Successfully tested locally (or docker image)
- [ x ] Successfully tested on a staging or sandbox server
- [ ] Successfully tested on a merchant's staging server

# Checklist:

- [ ] My code follows the style guidelines of this project.
- [ x ] I have performed a self-review of my own code.
- [ x ] I have commented my code, particularly in hard-to-understand areas.
- [ ] New and existing unit tests pass locally with my changes.
- [ ] I have created or modified unit tests to sufficiently cover my changes.
- [ ] I have added my Asana task link and provided a changelog message if applicable.

### CODE REVIEW

File: tests/load/setup_files/CartDataLoadTestController.php [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+     * containing cart with an array of cart items in
+     * format: {"cart":[{"id": 1} ... ]}
+     * Responds with order_token and order_reference
+     * of the order that is created in Bolt.
+     */
+    public function indexAction()
+    {
+        try {
+            // creates a cart in session
+            $cart_items = $this->getRequestData()->cart;
```
Comment (matt-thomason) [inline tests/load/setup_files/CartDataLoadTestController.php]: camelCase this please.

File: tests/load/setup_files/CartDataLoadTestController.php [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+            // OrderControllerTrait::createAction():
+            $boltpayCheckout = Mage::app()->getLayout()->createBlock('boltpay/checkout_boltpay');
+            /** @var Mage_Sales_Model_Quote $quote */
+            $quote = $boltpayCheckout->getSessionQuote(Bolt_Boltpay_Block_Checkout_Boltpay::CHECKOUT_TYPE_MULTI_PAGE);
+
+            // OrderControllerTrait::getCartData():
+            /** @var Bolt_Boltpay_Model_BoltOrder $boltOrder */
+            $boltOrder = Mage::getModel('boltpay/boltOrder');
+
+            $cart_data = $boltOrder->getCachedCartData($quote, Bolt_Boltpay_Block_Checkout_Boltpay::CHECKOUT_TYPE_MULTI_PAGE);
```
Comment (matt-thomason) [inline tests/load/setup_files/CartDataLoadTestController.php]: Same here

File: tests/load/setup_files/CartDataLoadTestController.php (hunk 1 above)
Comment (kajan-bolt) [reply tests/load/setup_files/CartDataLoadTestController.php]: Sounds good, made the change

File: tests/load/setup_files/CartDataLoadTestController.php (hunk 2 above)
Comment (kajan-bolt) [reply tests/load/setup_files/CartDataLoadTestController.php]: Made the change here as well

**Pull request merged**